packages/chart/docs/QuartileCandlestick.md

```diff
@@ -0,0 +1,105 @@
+# QuartileCandlestick
+
+```meta
+{
+    "source": "https://github.com/hpcc-systems/Visualization/blob/master/packages/chart/src/QuartileCandlestick.ts#L4",
+    "extends": "SVGWidget"
+}
+```
+
+QuartileCandlestick displays a five-number summary of a range of numeric values. The five-number summary is the minimum, first quartile, median, third quartile, and maximum.
+
+```sample-code
+import { QuartileCandlestick } from "@hpcc-js/chart";
+
+new QuartileCandlestick()
+    .target("target")
+    .columns(["Min","25%","50%","75%","Max"])
+    .data([100,250,350,400,500])
+    .render()
+    ;
+```
+
+_orientation_ can be used to render this chart either vertically or horizontally.
+
+_lineWidth_ and _candleWidth_ can be used to set the pixel width of the lines and the overall pixel width of the drawn portion of this chart respectively. _candleWidth_ also controls the height of the drawn portion while using 'horizontal' orientation.
+
+_edgePadding_ sets the pixel padding between the edges of the containing element and the minimum/maximum lines.
+
+_roundedCorners_ sets the pixel radius of the drawn lines.
+
+_upperTextRotation_ and _lowerTextRotation_ sets the degrees of rotation for the column labels and column values respectively.
```

Comment:
This is good - we should call out any changes from the default sample in all the examples like this!